cranelift/filetests/src/function_runner.rs

```diff
@@ -563,6 +623,47 @@ fn make_trampoline(name: UserFuncName, signature: &ir::Signature, isa: &dyn Targ
     func
 }
 
+/// Hostcall invoked directly from a compiled function body to test
+/// exception throws.
+extern "C" fn __cranelift_throw(
```

Comment:
`extern "C-unwind"` otherwise unwinding out of this is UB (you are skipping past the aborting landingpad due to using a different unwinding mechanism).

Reply:
Could you elaborate on this? I was under the impression that `C-unwind` had to do with the system unwinder, or whatever the default is for the platform. For this unwinder landing pads aren't invoked no matter what, so by the same logic `C-unwind` is also UB because landing pads still aren't executed. 

To me it seems that the soundness of this function relies on there not actually being any landing pads, which is something we'd have to uphold?